.mage/magefile.go

```diff
@@ -41,14 +28,38 @@ func init() {
 
 func All() {
 	mg.Deps(
-		mg.F(mgconvco.ConvcoCheck, "origin/main..HEAD"),
-		mggolangcilint.GolangciLint,
-		mggoreview.Goreview,
-		mgmarkdownfmt.FormatMarkdown,
-		mgyamlfmt.FormatYaml,
+		ConvcoCheck,
+		GolangciLint,
+		Goreview,
+		FormatMarkdown,
+		FormatYaml,
 	)
 	mg.SerialDeps(
-		mggo.GoModTidy,
+		GoModtidy,
 		mggitverifynodiff.GitVerifyNoDiff,
 	)
 }
+
+func FormatYaml() error {
+	return mgyamlfmt.FormatYamlCommand()
+}
+
+func GoModtidy() error {
+	return mggo.GoModTidy().Run()
+}
+
+func Goreview(ctx context.Context) error {
+	return mggoreview.Command(ctx, "-c", "1", "./...").Run()
+}
+
+func GolangciLint(ctx context.Context) error {
+	return mggolangcilint.LintCommand(ctx).Run()
+}
+
+func FormatMarkdown(ctx context.Context) error {
+	return mgmarkdownfmt.Command(ctx, "-w", ".").Run()
```

Comment:
Does this command have long flag names? :thinking: If so, I think we should switch. Makes it easier to read and get a feeling for what the command does.

Reply:
I agree, but unfortunately not: https://github.com/shurcooL/markdownfmt/blob/master/main.go#L23